mirage/config/NIRCam_zeropoints.list

```diff
@@ -1,18 +1,18 @@
 Filter Pupil Module Detector VEGAMAG ABMAG STMAG PHOTFLAM PHOTFNU Pivot_wave
-F070W CLEAR A NRCA1 26.16 26.45614182774024 27.001201986651644 5.748379794034092e-20 9.49663e-31 0.704
-F090W CLEAR A NRCA1 26.28 26.809122393581703 27.88994321167786 2.5354144673386703e-20 6.86084e-31 0.901
-F115W CLEAR A NRCA1 26.02 26.82114778745016 28.439373694749563 1.5285398688609284e-20 6.785270000000001e-31 1.154
-F140M CLEAR A NRCA1 25.11 26.236886799687156 28.280919201856925 1.7687178878418194e-20 1.162175e-30 1.404
-F150W2 CLEAR A NRCA1 27.23 28.495253771445945 30.917332884113748 1.5598868435783303e-21 1.45186e-31 1.671
-F150W CLEAR A NRCA1 25.89 27.11953817864898 29.308638503066177 6.863898418154277e-21 5.15479e-31 1.501
+F070W CLEAR A NRCA1 26.18 26.41421553992525 26.959275698836656 5.974698153409092e-20 9.870520000000001e-31 0.704
```

Comment:
This looks fine, I am curious do we keep this anywhere online? A rest API would be useful for something like this.

Reply:
I don't think this is kept anywhere else online. It's related to the pipeline's flux calibration, but it uses different units.